.docker/Dockerfile.cli

```diff
@@ -16,13 +16,22 @@ COPY composer.json composer.lock /app/
 COPY scripts /app/scripts
 
 # Run composer. Additional `rm`s can be added to reduce the image size, with diminishing returns.
-RUN composer install --no-dev \
+RUN composer install --no-dev --quiet \
   && rm -rf ~/.composer/cache \
   && rm -rf /app/web/core/tests \
   && rm -rf /app/web/modules/contrib/webform/tests
 
 # Place remaining files from repository, note exclusions in .dockerignore.
 COPY . /app
 
-# Internal drush aliases for the platform.
-RUN mv /app/vendor/govcms/scaffold-tooling/drupal/govcms.site.yml /app/drush/sites/
+# Set up standard Drush aliases.
+# @todo COPY can be moved upstream.
+COPY .docker/config/cli/ci.site.yml /etc/drush/sites/
+ARG LAGOON_SAFE_PROJECT
+RUN sed -i "s/%%LAGOON_SAFE_PROJECT%%/${LAGOON_SAFE_PROJECT}/g" /etc/drush/sites/ci.site.yml
```

Comment:
I do not understand what is happening here (I saw that yml file). Why is it called `ci.site.yml` and not something else ? Why `ci`? What is it used for?

Reply:
It's used in continuous integration (ci) processes.

Reply:
this looks like a file in production container that is used only in CI. If this is a CI only - should not it be added in CI job itself. Or can the name be generalised to be used with other environemnts.

The comment says `Set up standard Drush aliases`  but I do not see how `ci.site.yml` is a standard alias. Something does not fit here.

Reply:
> The comment says `Set up standard Drush aliases` but I do not see how `ci.site.yml` is a standard alias. Something does not fit here.

Oh yeah sorry about this, I made a mistake when copying updates from my test project. This is the contents below which will be pushed soon. It's basically a rename of govcms.site.yml which i found too generic when i discovered it was only used in CI. I expect the name will be reviewed when srowlands has a chance to review the PR.

```
##
# File /etc/drush/sites/ci.site.yml
# @todo move upstream, see Dockerfile.cli
#
# This is used in integration and workflow processes in Openshift and GitLab.
# It is not intended to be used by clients.
# Run with `drush --alias-path=/etc/drush/sites ci.ALIASNAME` 
#

prod:
  root: /app
  host: ssh.lagoon.svc
  user: %%LAGOON_SAFE_PROJECT%%-master
  uri: nginx-%%LAGOON_SAFE_PROJECT%%-master.govcms.amazee.io
  ssh:
    options: -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -p 2020 -F /dev/null
```

Reply:
Re-added (and tested the build locally)